src/commands/BreakCommand.java

```diff
@@ -66,9 +67,14 @@ public void execute(String[] input, SimulationManager simulationManager) {
             String allConstraints = String.join(" ", Arrays.copyOfRange(input, 1, input.length));
             Matcher m = Pattern.compile(CommandConstants.CONSTRAINT_REGEX).matcher(allConstraints);
 
+            AliasManager am = simulationManager.getAliasManager();
             while (m.find()) {
                 String constraint = m.group(1).replace("\"", "");
 
+                if (am.isAlias(constraint)) {
+                    // TODO (liangdrew)
```

Comment:
Will this be implemented in this PR? The issue that this PR closes mentions the `break` command, so it makes sense to include the implementation here as well.

Reply:
This and tests for it will follow in another PR. I'd prefer to separate the work this way to keep the PRs smaller.